Workflow.Rmd

```diff
@@ -368,85 +370,351 @@ state_accounts_index %>%
+
+```{r, echo = T}
+mini_loans %>% 
+  ggplot(aes(x = revol_util, y = defaulted)) +
+  geom_jitter(alpha = 0.01, height = 0.05) +
+  geom_smooth() +
+  theme_bw() +
+  labs(title = "Probability of default by utilization ratio") +
+  xlim(0, 100)
+```
+
+This tells us something we might already expect: that those who are using more of their available credit lines
+are far more likely to default than those who are not. 
+
+Above we've performed a quick exploratory analysis of a real-life commercial dataset. We've learned that 
+credit scores and loan purposes are both strongly associated with default risk, as is a high revolving utilization 
+rate, and a low income. 
+
+Now we're on our way to building a model, but not before building an understanding of the bits and pieces that go 
+into building a good model. The next secion describes these. 
+
+\newpage 
+
+## Part 3: A gentle introduction to required knowledge for modeling 
+
+This book is about modeling, and so it makes sense to describe what we think of as being a model. By thinking about models
+holistically, you'll find that you can move between domain areas with relative ease. The concepts in this section
+are just as applicable to economics as they are to pharmacometrics or natural language processing or computer vision. It's 
+all essentially the same! Fundamentally, any model has the following: 
+
+- A collection of "knowns"---variables that we treat as being known for sure, or assume to take
+particular values. If we're building a predictive model of whether a borrower defaults on a loan, this might be the information
+we have about the customer at the time we will be called on to make predictions. If we're a government choosing a fiscal policy, 
+it might be a projected spending pattern (even though this is not "known", for a modeling exercise we might pretend it is known
+for the sake of some policy simulation). In some cases, especially in time series and computer vision, 
+the set of "knowns" might be empty. We often use $X$ for "knowns". 
+- Some "model unknowns", normally called $\theta$. These might be latent variables, coefficients, scales, weights, 
+etc. Throughout this book we'll use the term "model unknown" rather than the more common "parameter", which we'll reserve
+for a specific use. In principal, we can build any model and plonk values in place of these model unknowns, and perform
+some analysis (thought that analysis might be bunk). In fact, many applications of economics---particularly 
+computable general equilibrium modeling---involve "calibrating" models in precisely this way, using 
+"reasonable" assumed values for model unknowns. Poorly chosen values
+of these unknowns will typically make the model say useless things, and most of this book is about strategies to come up
+with good values for these unknowns. 
+- A "model structure", often denoted by $\mathcal{M}$. This is a mathematical structure that provides some mapping between 
```

Comment:
I like this explanation a lot but I think people might not understand it right away (i.e. the sense in which a neural network is a "model" is not super intuitive). That might be fine, I'm torn.